api/test/service/store.test.js

```diff
@@ -174,21 +171,9 @@ test('store remove does not fail for non existent link', async (t) => {
 
   // expect no response for a remove
   t.falsy(storeRemove)
-
-  const storeRemove2 = await StoreCapabilities.remove.invoke({
```

Comment:
this looks not needed added in previous PR?

Reply:
cc @olizilla

Reply:
no it was deliberate. I wanted the test to show that trying to remove the same thing twice works.

Reply:
same non existent thing? not sure I understand what happens differently. Perhaps you wanted to remove after a successful remove?

Reply:
I see "dont error if delete called on missing thing" as "be idempotent". This test verifies that we can call remove twice with the same cid and get the same result.